test/src/test/java/org/corfudb/infrastructure/ServerContextBuilder.java

```diff
@@ -14,14 +15,11 @@
 // Disable magic number check to make defaults readable
 @SuppressWarnings("checkstyle:magicnumber")
 public class ServerContextBuilder {
-
-    long initialToken = 0L; // for testing, we want to reset the sequencer on each test
-
     boolean single = true;
     boolean memory = true;
     String logPath = null;
-    boolean noVerify = false;
-    boolean noSync = false;
+    boolean verifyChecksum = true;
+    boolean syncData = false;
```

Comment:
would the default be true? to be consistent with the previous default?

Reply:
This is  just to make tests go faster. I think fsync behavior is hard to test from these unit tests.